development.md

```diff
@@ -1,85 +1,162 @@
+# Improve development guide 
+
 # Development setup
 
-This is a guide to running the Zulip desktop app from a source tree,
-in order to contribute to developing it.
+This is a guide to running the Zulip desktop app from a source tree, in order to contribute to developing it. The Zulip electron development environment can be installed on **macOS, Windows, and Linux** (Debian or Ubuntu recommended). You’ll need at least **2GB of available RAM**. Installing the Zulip electron development environment requires downloading several hundred megabytes of dependencies, so you will need an **active, reasonably fast, internet connection throughout the entire installation processes.**
+
+# Set up Git & GitHub
+
+You can skip this step if you already have Git, GitHub.
+
+Follow our [Git Guide](https://zulip.readthedocs.io/en/latest/git/setup.html) in order to install Git, set up a GitHub account
+
+
+# Install Prerequisites
+
+Jump to:
+
+- [MacOS](https://paper.dropbox.com/doc/Improve-development-guide--AaFMtIWoCHF1TcoqW7Xz_ARdAg-IOXUYkXRC2dLnSmvuk13t#:h2=MacOS)
```

Comment:
You need to update paper link to GitHub.

Reply:
oh didn't noticed :sweat_smile:. fixing right now

Reply:
updated the links, you can review